e2e/tests/nightly/notebook/notebook-upload-file.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('Notebook Upload File Test', () => {
   const workspaceName = 'e2eTestNotebookUploadFile';
   const pyNotebookName = makeRandomName('Py3');
   const pyFileName = 'nbstripoutput-filter.py';
-  const pyFilePath = path.relative(process.cwd(), __dirname + `../../../resources/python-code/${pyFileName}`);
```

Comment:
is it possible ot change this to use an absolute path?

Reply:
Yeah, it should be. I went with the fastest fix for now, though.

Reply:
I considered this but wanted to returned to [RW-10342](https://precisionmedicineinitiative.atlassian.net/browse/RW-10342) immediately - I think there are bigger E2E improvements to work on now 😄

Reply:
I had considered this but wanted to returned to [the E2E spike](https://precisionmedicineinitiative.atlassian.net/browse/RW-10342) immediately - I think there are bigger E2E improvements to work on now 😄

Reply:
I had considered this but wanted to return to [the E2E spike](https://precisionmedicineinitiative.atlassian.net/browse/RW-10342) immediately - I think there are bigger E2E improvements to work on.